content/bn/docs/reference/glossary/uid.md

```diff
@@ -0,0 +1,17 @@
+---
+title: UID
+id: uid
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/names
+short_description: >
+  কুবেরনেটিস সিস্টেম দ্বারা তৈরি একটি স্ট্রিং যা অবজেক্টকে অনন্যভাবে চিহ্নিত করে।
+
+aka: 
+tags:
+- fundamental
+---
+কুবেরনেটিস সিস্টেম দ্বারা তৈরি একটি স্ট্রিং যা অবজেক্টকে অনন্যভাবে চিহ্নিত করে।
```

Comment:
Kindly use `কুবারনেটিস` instead of `কুবেরনেটিস` and `ইউনিকভাবে` instead of `অনন্যভাবে`.